[ADD] project_task_type_remaining_time_zero

victor-ditay opened the pull request:
- Forgoten feature on refactoring elico_project

Commit:
[ADD] project_task_type_remaining_time_zero
- Forgoten feature on refactoring elico_project

victormartinelicocorp commented:
@elicoidal @seb-elico Please have a check

elicoidal commented on project_task_type_remaining_time_zero/README.rst:
```diff
@@ -0,0 +1,54 @@
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/Elico-Corp/{project_repo}/issues>`_. In case of trouble, please
```
update

elicoidal commented on project_task_type_remaining_time_zero/__init__.py:
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (https://www.elico-corp.com)
```
remove copyright in `__init__.py`

elicoidal reviewed (approved):
LGTM

Commit:
[FIX] project_task_type_remaining_time_zero
- remove copyrights on init py
- add the right repository on readme file

victormartinelicocorp commented:
@elicoidal

elicoidal commented:
Can you check travis?

Commit:
[FIX] project_task_type_remaining_time_zero
- Fix missing key on manifest file

Commit:
[FIX] project_task_type_remaining_time_zero
- Pylint

victormartinelicocorp commented:
> Can you check travis?

```
109.77s$ travis_run_tests

======== Testing test_flake8 ========

project_completion_report/__openerp__.py:25:1: W391 blank line at end of file

project_completion_report/report/project_completion_report.py:45:9: E128 continuation line under-indented for visual indent

project_completion_report/report/project_completion_report.py:163:1: W391 blank line at end of file

======== Testing test_pylint ========

************* Module auth_cas.res_config

auth_cas/res_config.py:10: [W0403(relative-import), ] Relative import 'pycas', should be 'auth_cas.pycas'

From https://github.com/Elico-Corp/odoo-addons

 * [new branch]      8.0        -> origin/8.0
```

@elicoidal I fixed the errors regards this module, but basic all the other modules already on the master branch would fail.
Let fix on an specific PR as actually I can't know how many of them would fail

elicoidal commented:
I thought you already fixed the issue of pycas

victormartinelicocorp commented:
> I thought you already fixed the issue of pycas

@elicoidal I couldn't find time. Think now would finish it.

victormartinelicocorp commented:
> I thought you already fixed the issue of pycas

@elicoidal One thing is that I haven't spotted yet:

This would implies migrate to new API all modules pointed.
So is more than just small adjustment on Travis. Not a problem for me as far you know.

cc @seb-elico

elicoidal commented:
@victormartinelicocorp IMO no need for new API in v8 (not mandatory by Travis).
Fixing the pycas import should be good enough

Commit:
Merge branch '8.0' of github.com:Elico-Corp/odoo-addons into 8.0-ADD-project_remaning_time_zero_feature

victormartinelicocorp commented:
> @victormartinelicocorp IMO no need for new API in v8 (not mandatory by Travis).
Fixing the pycas import should be good enough

@elicoidal Weird that was pointed by Travis, I really didn't want to modify those modules but start to raise errors for the non CamelCase class, use of cr.commit(), old api osv deprecated and other like files should not have execution rights.

seb-elico commented:
@victormartinelicocorp This module has nothing to do with task **type**. It should be called `project_task_stage_remaining_time_zero`. Please refactor all references to `type`

seb-elico reviewed (changes requested):
Refactor `type` to `stage`

seb-elico commented:
@victormartinelicocorp FYI Travis is failing for all the repo, not just your PR. I think it's better to create a separate PR to fix Travis, later this one will become green automatically (might require a re-build though)

victormartinelicocorp commented:
> Refactor type to stage

@seb-elico Odoo but internally is called `type` but agree the name is not so good so I will change it.

victormartinelicocorp commented:
> @victormartinelicocorp FYI Travis is failing for all the repo, not just your PR. I think it's better to create a separate PR to fix Travis, later this one will become green automatically (might require a re-build though)

@seb-elico Is what I did (#196) then I refactored this PR. Is still failing on which PR?

seb-elico commented:
> @seb-elico Odoo but internally is called type but agree the name is not so good so I will change it.

@victormartinelicocorp Well spotted! Indeed functional name is "Task Stage" and technical name is `project.task.type`.
@elicoidal Which name seems the best? Maybe call the module "Project Task Stage - Set remaining time to zero" and keep the technical name `project_task_type_remaining_time_zero`

> @seb-elico Is what I did (#196) then I refactored this PR. Is still failing on which PR?

Maybe you need to re-build this one now (simply add 1 commit)

Commit:
[FIX] project_task_stage_remaining_time_zero
- Improve name of module as would be better to understand
over which feature is.

elicoidal commented:
> Which name seems the best? Maybe call the module "Project Task Stage - Set remaining time to zero" and keep the technical name project_task_type_remaining_time_zero

What about: project_task_set_remaining_time_zero

elicoidal commented:
What are we waiting for?

victormartinelicocorp commented:
> What are we waiting for?

@seb-elico Could you check to approve it?
